### PR TITLE
(build): Exclude Cesium from minification

### PR DESCRIPTION
**Pull request by les2**
Closes https://github.com/ember-cli/ember-cli/issues/6594

### CODE REVIEW

File: ember/ember-cli-build.js [hunk 1]
```diff
@@ -10,6 +10,10 @@ module.exports = function(defaults) {
       extensions: ['css', 'js'],
       exclude: ['cesium'],
     },
+    minifyJS: {
+      enabled: EmberApp.env() !== 'production',
```
Comment (Turbo87) [inline ember/ember-cli-build.js]: do we have to add the `enabled` key here or will it default to that automatically?

Comment (les2) [inline ember/ember-cli-build.js]: The docs are not clear. It could be either way. I'll test it and see.

Comment (les2) [inline ember/ember-cli-build.js]: Interesting. I just ran a test locally by deleting the `enabled` flag completely.

The result was that the `exclude` setting was completely ignored! Only when the `enabled` key is present - it seems - is the `exclude` setting honored.

This is perhaps a bug with ember-cli or at the very least a documentation issue.

I would actually like for someone else to confirm this for me as this is kinda unbelievable. :)

# Summary -> You need the `enabled` key for any settings to take effect, I think.

Comment (Turbo87) [inline ember/ember-cli-build.js]: okay, thanks for looking into this! I'll merge this as is for now and will follow this up later in Ember CLI.

Comment (Turbo87) [inline ember/ember-cli-build.js]: hmm, I just noticed, shouldn't the check be `EmberApp.env() === 'production'` instead of `!==`?

Comment (les2) [inline ember/ember-cli-build.js]: holy crap! i guess i'm trolling! i think i copied that from ember-cli website. let's see.

screen shot 2017-03-29 at 2 42 28 pm

# I copied from the sourcemap default section!

you are correct!

Comment (les2) [inline ember/ember-cli-build.js]: retesting.

**Comment by les2** [comment]
@Turbo87 closing while i double-check everything. i tested this locally on a dummy project and confirmed that the options work.

**Comment by les2** [comment]
@Turbo87 sorry for the confusion.

1. The `enabled` flag is not necessary so I removed that.
2. The `exclude` option for uglify required nesting under the `options: {...` key

Testing
=======

Use this script to run build:
```
#!/usr/bin/env bash
rm -rf "${TMPDIR}/if-you-need-to-delete-this-open-an-issue-async-disk-cache" && rm -rf tmp && rm -rf dist && ember build --environment production
```

- With the changes to ember-cli-build.js, we do not get that warning about compilation time of Cesium.js.
- With `out` the changes to ember-cli-build.js, we do get the error about compilation time.

**Comment by les2** [comment]
gonna rebase those fixup commits out now.

**Comment by Turbo87** [comment]
awesome, thanks!

**Pull request merged**